apps/server/internal/httpapi/api_test.go

```
@@ -0,0 +1,273 @@
+	cases = append(cases,
+		struct { name, method, path, body string; status int }{"list issues", "GET", "/api/projects/" + projectID + "/issues", "", 200},
+		struct { name, method, path, body string; status int }{"create issue", "POST", "/api/projects/" + projectID + "/issues", issueBody, 201},
+		struct { name, method, path, body string; status int }{"get issue", "GET", "/api/projects/" + projectID + "/issues/" + issueID, "", 200},
+		struct { name, method, path, body string; status int }{"update issue", "PATCH", "/api/projects/" + projectID + "/issues/" + issueID, `{"status":"IN_PROGRESS"}`, 200},
+		struct { name, method, path, body string; status int }{"assign issue", "POST", "/api/projects/" + projectID + "/issues/" + issueID + "/assignment", `{"agentId":"` + agentID + `"}`, 202},
+		struct { name, method, path, body string; status int }{"list runs", "GET", "/api/projects/" + projectID + "/runs", "", 200},
+		struct { name, method, path, body string; status int }{"get run", "GET", "/api/projects/" + projectID + "/runs/" + runID, "", 200},
+	)
+	for _, tc := range cases {
+		t.Run(tc.name, func(t *testing.T) {
+			req := httptest.NewRequest(tc.method, tc.path, strings.NewReader(tc.body))
```

golangci-lint (2.13.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)